src/main/java/com/dougnoel/sentinel/elements/tables/Table.java

```diff
@@ -87,7 +87,7 @@ protected List<String> getOrCreateHeaders() {
 		if (headers.isEmpty()) {
 			getOrCreateHeaderElements();
 			for (WebElement header : headerElements) {
-				String headerText = header.getText().replaceAll("[\\t\\n\\r]+"," ");
+				String headerText = header.getText().replaceAll("[\\t\\n\\r]+"," ").strip();
```

Comment:
```suggestion
				String headerText = header.getText().replaceAll("[\\t\\n\\r]+","");
```

Reply:
Attempted to remove the space and this caused many tests to fail.